backends/platform/sdl/win32/win32.mk

```diff
@@ -85,7 +85,11 @@ endif
 endif
 
 win32dist-mingw: win32-data
+ifeq ($(LDD),ldd)
```

Comment:
I would more revert the logic here as ldd is the standard tool and peldd a specific one.
Main problem here is that you use the same variable for 2 incompatible programs

Reply:
Done. Can you suggest a better alternative? Different target maybe?